panther_power_control/src/power_board_node.py

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
I am unable to add a comment in line [267](https://github.com/husarion/panther_ros/blob/37319304dbb76bac243e2d541b75d5f5e6522c0b/panther_power_control/src/power_board_node.py#L267), but I believe that we can use '_lock' there. It is currently functioning, so the decision is up to you. The same situation applies to `_clearing_e_stop`.